src/components/Geo/index.jsx

```diff
@@ -1,10 +1,36 @@
 import React from 'react'
+import ReactDOM from "react-dom";
 import { Helmet } from 'react-helmet';
 import MapPlot from './MapPlot';
 import SelectionInfo from './SelectionInfo';
+import {
+    ExternalLink,
+    helpIcon
+} from '../../common/Helpers';
+import styles from '../../styles/collapse.css';
 
-const Geo = () => {
+function Details({ defaultOpen = false, title, children }) {
+  function handleKey({ key }) {
+    [" ", "return"].includes(key) && setOpen(o => !o);
+  }
+  const [open, setOpen] = React.useState(defaultOpen);
+  console.log(open);
+  return (
+    <details
+      open={defaultOpen}
+      onKeyDown={handleKey}
+      onClick={() => setOpen(o => !o)}
+    >
+      <summary>{title}</summary>
+      {open && children}
+    </details>
+  );
+}
+
+const Geo = ({ collapsed, children }) => {
```

Comment:
`{ collapsed, children }` denote “props” that would be defined by the parent (in this case [App.jsx](https://github.com/serratus-bio/serratus.io/blob/a862c4d14a4b81d1a760ef383226dfd38c3c9833/src/App.jsx#L38)). They aren’t used and should be removed.